spec/services/stats/management_information_generator_spec.rb

```diff
@@ -22,8 +22,8 @@
       expect(contents.size).to eq(valid_claims.size + 1)
     end
     
-    it 'has 19 columns' do
-      expect(contents.first.split(',').count).to eql 19
+    it 'has 21 columns' do
```

Comment:
Not you, i know but it might be clearer and more useful to have something like

```
let(:report_columns) { [whatever whatever1] }

it 'has expected columns' do
  expect(contents.first.split(',').to match_array(report_columns)
end
```

Reply:
👍 have added it 'has expected columns' test to the spec.